job-service-contract/src/main/resources/com/hpe/caf/services/job/swagger.yaml

```diff
@@ -218,6 +217,36 @@ paths:
           description: The `jobId` parameter contains invalid characters.
         503:
           description: The request failed due to a database connection error.
+  /partitions/{partitionId}/jobs/{jobId}/status:
+    parameters:
+      - $ref: '#/parameters/partitionId'
+      - name: CAF-Correlation-Id
+        in: header
+        type: string
+        description: |
+          An identifier that correlates events 
+          across different services.
+      - $ref: '#/parameters/jobId'
+    get:
+      tags:
+        - Jobs
+      summary: Gets the status of the job
+      description: Gets the status of the specified job.
+      operationId: getJobStatus
+      responses:
+        200:
+          schema:
+            $ref: "#/definitions/job-status"                
```

Comment:
The new status endpoint just returns a string in the response, e.g. "Active", "Cancelled" etc. It does not wrap this in a JSON object like:

```
{
  "status": "Active"
}
```

I think this is ok, it's consistent with the 2 other endpoints here that just return a single value (the `isActive` and `jobstats/count` endpoints), and results in a bit less boilerplate code around wrapping the response and unwrapping it in the client code etc.

I believe its fine from a JSON validity point of view:

https://stackoverflow.com/questions/7487869/is-this-simple-string-considered-valid-json